smart_campus/app/templates/app/station_edit.html

```diff
@@ -0,0 +1,159 @@
+          <input id="name" name="name" type="text" placeholder="站點名稱" class="form-control input-md" value="{{form_data.name}}" required="">
+        </div>
+      </div>
+
+      <!-- Category Select -->
+      <div class="col-md-4">
+        <label class="control-label col-md-12" for="select_category">類別</label> 
+        <select id="select_category" name="category" class="custom-select" required="">
+          <option value>請選擇類別...</option>
+          {% if categories %}
+            {% for category in categories %}
+              <option value="{{category.name}}" {% if form_data.category|stringformat:"s" == category.name|stringformat:"s" %}selected{% endif %} >{{category.name}}</option>
+            {% endfor %}
+          {% endif %}
+        </select>
+      </div>
+    </div>
+    <br>
+
+    <!-- Content Textarea -->
+    <div class="row">
+      <div class="col-md-6">
+        <label class="col control-label" for="content">推播內容</label>
+        <div class="col">                     
+          <textarea class="form-control" id="content" name="content">{{form_data.content}}</textarea>
+        </div>
+      </div>
+    </div>
+    <br>
+
+    <!-- Button : Change Image -->
+    <div class="row" id="img_change">     
+      <div class="col-md-6">
+        <label class="col control-label">更改圖片</label>
+        <button type="button" class="btn btn-danger col-md-6">更改上傳圖片</button>
+        <input type="text" name="img_changed" id="img_changed" value='false' hidden>
+      </div>
+    </div>
+    <br>
+    <!-- Image Upload Section : HIDDEN-->
+    <div class="row" id="img_upload_sec" style="display: none;">
+      <div class="col-md-4" id="img_upload">
+        <label class="col-md-4 control-label">圖片上傳</label>
+        <div class="col-md-4">
+          Img 1:
+          <input id="img1" name="img1" class="form-control-file" type="file" accept="image/*">
+        </div>
+      </div>
+      <div class="col">
+        <br>
+        <button type="button" id="btn_addimage" name="btn_addimage" class="btn btn-danger" onclick="">新增照片欄位</button>
+      </div>
+    </div>
+    <br>
+
+    <!-- Choose Primary photo : HIDDEN-->
+    <div class="row" id="img_choose_sec" style="display: none;">
+      <div class="col-md-8" id='radios-img-select'> 
+        <label class="col-md-3 control-label" >選擇主要照片</label>
+        <label class="radio-inline" for="radios-1">
+          <input type="radio" name="main_img_num" id="radios-1" value="1" checked="checked">
+          1
+        </label>  
+      </div>
+    </div>
+    <br>
+
+    <!-- Beacon Information -->
+    <div class="row">
+      <div class="col-md-5">
+        <div class="col-md-12">
+
+          <!-- Select linking Beacon from Map -->
+          <label class="control-label col" for="selectbasic">Beacon選擇(請在地圖點選)</label> 
+          <select id="selectbasic" name="beacon" class="custom-select" required="">
+            <option value>請在地圖點選...</option>
+            {% if beacons %}
```

Comment:
It seems to me this `if` block is the same as line 34~38.
Please rewrite it as another tempalte.